tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/visitor/TreeVisitor.java

```diff
@@ -276,6 +276,46 @@ public void visit(Comment comment) {
         ballerinaASTAPI.addComment(comment.getComment());
     }
 
+    /**
+     * Prints the logger message in correct log level. In mule, if the message is not set with any value it prints
+     * out the whole message property details. Since in Ballerina, this is not directly available that is not
+     * provided here.
+     *
+     * @param log
+     */
+    @Override
+    public void visit(org.wso2.ei.tools.mule2ballerina.model.Logger log) {
```

Comment:
Any reason for using fully qualified classes here ?

Reply:
This Logger represents a mule logger and since the slf4j Logger has been already imported, have to use the fully qualified name for this Logger.